Split getting started into SGX and simulation mode (#72)
edgelesssys/marblerun.sh: content/docs/getting-started/quickstart-sgx.md

```

You can check that the state of Marblerun changed and is now ready to authenticate your services by running:

```bash
marblerun status $MARBLERUN
```

Create and annotate the emojivoto namespace for [auto-injection]({{< ref "docs/features/auto-injection.md" >}}):

```bash
kubectl create namespace emojivoto
marblerun namespace add emojivoto
```

### Step 3.2: Deploy emojivoto

Finally, install the demo application onto your cluster.
Please make sure you have [Helm](https://helm.sh/docs/intro/install/) ("the package manager for Kubernetes") installed at least at Version v3.2.0.
Install emojivoto into the emojivoto namespace by running:

```bash
helm install -f ./kubernetes/sgx_values.yaml emojivoto ./kubernetes -n emojivoto
```

## Step 4: Watch it run

You can now check the Marblerun log and see the services being authenticated by the Coordinator.

```bash
kubectl -n marblerun logs -f -l edgeless.systems/control-plane-component=coordinator
```

Port forward the front-end web service to access it on your local machine by running:

```bash
sudo kubectl -n emojivoto port-forward svc/web-svc 443:443 --address 0.0.0.0
```

Now visit [https://localhost](https://localhost).

> You'll be presented with a certificate warning because your browser does not know Marblerun's root certificate as a root of trust.

Reviewer: ```suggestion
You'll be presented with a certificate warning because your browser does not trust Marblerun's self-signed root certificate by default.
```

> You can safely ignore this error for now and proceed to the website.\

Reviewer: Maybe we can add here that we actually attested this and thus, we can trust it somehow? Of course the browser does not do it by default (and web attestation is a bit difficult to achieve), but I guess it would make sense to explain _why_ we can ignore this _for now_.

Author: The easy fix would be to install the Marbleruns root certificate in the browser (which is done during the full demo), but this goes slightly above the scope of a quickstart imo
